scopes/design/ui/alert-card/alert-card.module.scss

```diff
@@ -0,0 +1,22 @@
+.card {
+  padding: 24px;
+}
+
+.heading {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.title {
+  font-family: 'CircularPro';
```

Comment:
please remove. if it still stays when you replace `h6` with `div`, set it to `font-family: inherit`

Reply:
I believe this is needed if we are going to use a <H6> which would be more semantically correct.